blankdiscordbot-core/src/main/resources/application.yml

```diff
@@ -19,6 +22,22 @@ web3:
     salesChannel: 934187617895997440
     contractWatchList:
       "0x881d9c2f229323aad28a9c9045111e30e1f1eb25": "Initializer"
+  nfts:
+    contracts:
+      - "0x881d9c2f229323aad28a9c9045111e30e1f1eb25" 
+emotes:
+  size: medium
+  initializers:
+    - GM
```

Comment:
I'm glad these are configured, will want to start with GM, GN, lovesThis, and sendsLove only